### PR TITLE
Workflow Automated Test - LRQA-67558 | master

### PR DESCRIPTION
**Pull request by brunadmelo**

### CODE REVIEW

**Comment by brunadmelo** [comment]
https://issues.liferay.com/browse/LRQA-67558, cc:@rodrigocunhaa

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsWorkloadByStep.testcase [hunk 1]
```diff
@@ -57,12 +57,56 @@ definition {
...
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			panel = "Site Administration",
+			portlet = "Blogs");
+
+		Blogs.addWithWorkflow(
+			entryContent = "Blogs Entry Content",
```
Comment (CalumR23) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsWorkloadByStep.testcase:95]: Could you please change this macro to 
`JSONBlog.addEntry(
				entryContent = "Blogs Entry Content",
				entryTitle = "Blogs Entry Title");`

Comment (brunadmelo) [reply modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsWorkloadByStep.testcase:95]: Ok, i'll change and resend.

**Comment by brunadmelo** [comment]
I changed what was requested and the final part of the test to match the steps updated in the ticket (Jira).

**Comment by CalumR23** [comment]
[Sent to BChan](https://github.com/CalumR23/liferay-portal/pull/10)

**Pull request closed without merging**